pkg/compose/build_buildkit.go

```diff
@@ -46,6 +48,12 @@ func (s *composeService) doBuildBuildkit(ctx context.Context, project *types.Pro
 	defer cancel()
 	w := xprogress.NewPrinter(progressCtx, os.Stdout, mode)
 
+	var servicesName []string
+	for name := range opts {
+		servicesName = append(servicesName, name[strings.LastIndex(name, "_")+1:])
```

Comment:
Isn’t the separator `"-"` or `"_"` depending on `--compatibility`?

Reply:
indeed, see my earlier comment on this being a fragile approach